__tests__/app.test.js

```diff
@@ -32,23 +32,53 @@ describe("GET", () => {
             expect(topic).toHaveProperty("description");
           });
         });
+      });
+    test("STATUS - 404 respond with a message bad request when people request with an invalid endpoint", () => {
+      return request(app).get("/api/topicafjajfh").expect(404);
     });
-    test("STATUS - 400 respond with a message bad request when people request with an invalid endpoint", () => {
-        return request(app)
-        .get("/api/topicafjajfh")
-        .expect(404);
-    })
   });
-  describe('/api', () => {
-    test('STATUS - 200, should respond with all the list of endpoints available ', () => {
-        return request(app)
+  describe("/api", () => {
+    test("STATUS - 200, should respond with all the list of endpoints available ", () => {
+      return request(app)
         .get("/api")
         .expect(200)
-        .then(({body}) => {
-          console.log(body)
-          const bodyLength = Object.keys(body).length
-          expect(bodyLength).toBe(3)
-          expect(body).toEqual(endpoints)
+        .then(({ body }) => {
+          expect(body).toEqual(endpoints);
+        });
+    });
+  });
+  describe("/api/articles/:article_id", () => {
+    test("STATUS - 200, should respond with array of article objects", () => {
+      return request(app)
+        .get("/api/articles/2")
+        .expect(200)
+        .then((response) => {
+          const { article } = response.body
+          expect(article).toHaveLength(1);
+          expect(article[0]["article_id"]).toBe(2);
+          expect(article[0]).toHaveProperty('title');
+          expect(article[0]).toHaveProperty('author');
+          expect(article[0]).toHaveProperty('body');
+          expect(article[0]).toHaveProperty('topic');
+          expect(article[0]).toHaveProperty('created_at');
+          expect(article[0]).toHaveProperty('votes');
+          expect(article[0]).toHaveProperty('article_img_url');
+        });
```

Comment:
As we are only dealing with 1 article object, let's make it airtight and test the whole object instead of just the properties 🙂